daemons/fenced/pacemaker-fenced.c

```diff
@@ -1623,8 +1633,8 @@ main(int argc, char **argv)
             rc = pcmk__add_logfile(*fname);
 
             if (rc != pcmk_rc_ok) {
-                fprintf(stderr, "Logging to %s is disabled: %s\n",
-                        *fname, pcmk_rc_str(rc));
+                out->err(out, "Logging to %s is disabled: %s",
```

Comment:
I did this here and also in one of the other daemons (I think execd) in a previous PR. It might need a tweak in hindsight... not sure that we're guaranteed to see the message until after the daemon exits. Maybe stick with `fprintf()` or call `out->finish()`

Reply:
Not worth caring about. It would only make a difference for XML output, which is unlikely to be used in combination with -l (which is only used by cts-fencing in practice).